Stratis.Bitcoin.Features.Notifications/BlockNotification.cs

```diff
@@ -76,57 +78,73 @@ public virtual void SyncFrom(uint256 startHash)
         /// <summary>
         /// Notifies about blocks, starting from block with hash passed as parameter.
         /// </summary>
-        public virtual IAsyncLoop Notify()
+        public void Start()
         {
-            return this.asyncLoopFactory.Run("block notifier", token =>
-            {
-                // Not syncing until the StartHash has been set.
-                if (this.StartHash == null)
-                {
-                    return Task.CompletedTask;
-                }
+            this.asyncLoop = this.asyncLoopFactory.Run("Notify", async token =>
+           {
+               await Notify(this.nodeLifetime.ApplicationStopping);
+           },
+            this.nodeLifetime.ApplicationStopping,
+            TimeSpans.Ms100);
+        }
```

Comment:
@dangershony Can you perhaps confirm that the "repeat every" time frame is correct here?

Reply:
No it previously was 1000 ms. (default)